lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
This looks somewhat similar to [`embed_tokens()`](https://github.com/mishoo/UglifyJS2/blob/642273c29002e7676719f489d7fcf552974118f4/lib/parse.js#L789) which is use extensively throughout this file. Would you mind considering whether we can reuse that in this case as well?